src/Api/Checkout.php

```diff
@@ -0,0 +1,162 @@
+<?php namespace Lean\Woocommerce\Api;
+
+use Lean\AbstractEndpoint;
+use Epoch2\HttpCodes;
+use Lean\Woocommerce\Utils\ErrorCodes;
+use Lean\Woocommerce\Utils\Hooks;
+
+/**
+ * Class Checkout.
+ *
+ * @package Lean\Woocommerce\Api
+ */
+class Checkout extends AbstractEndpoint
+{
```

Comment:
I think we should follow a consisten bracket style and the WP style guide follows the style used on the methods on the same line as the class.